### PR TITLE
Solved: On screen rotation selected forms get unselected #120

### PR DESCRIPTION
**Pull request by HarshitSachan1999**
Closes #

#### What has been done to verify that this works as intended?
I checked it by running the app in android device.

#### Why is this the best possible solution? Were any other approaches considered?
I found it best solution as it includes only one line of code in manifest file and it also solves bug #247. Yes, firstly I tried to save the changes within activity file but that wasn't working.

#### How does this change affect users? Describe intentional changes to behavior and behavior that could have accidentally been affected by code changes. In other words, what are the regression risks?
I am not finding any regression risk.

#### Before submitting this PR, please make sure you have:
- [x] run `./gradlew checkCode` and confirmed all checks still pass OR confirm CircleCI build passes
- [x] verified that any code or assets from external sources are properly credited in comments and/or in the [about file](https://github.com/opendatakit/share/blob/master/share_app/src/main/assets/open_source_licenses.html).

### CODE REVIEW

**Comment by HarshitSachan1999** [comment]
@Chromicle I had't seen #248, what should I do now.

File: skunkworks_crow/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -83,7 +83,8 @@
 
         <activity
             android:name=".views.ui.send.SendFormsActivity"
-            android:parentActivityName=".views.ui.main.MainActivity">
+            android:parentActivityName=".views.ui.main.MainActivity"
+            android:configChanges="keyboardHidden|orientation|screenSize">
```
Comment (lakshyagupta21) [inline skunkworks_crow/src/main/AndroidManifest.xml]: @HarshitSachan1999  we should not handle screen orientation changes using manifest changes.

**Comment by lakshyagupta21** [comment]
Closing this PR, @HarshitSachan1999 please go through the Github issue once you find any suitable issue and check if there are any open PR references or not, before start working on any issue. I know @opendatakit-bot is behaving very weird nowadays and we'll try to rectify the issue.

File: skunkworks_crow/src/main/AndroidManifest.xml (hunk 1 above)
Comment (HarshitSachan1999) [reply skunkworks_crow/src/main/AndroidManifest.xml]: @lakshyagupta21  Does this mean that I have to look for some other method to resolve it ?

**Pull request closed without merging**